ui/lib/core/addon/components/search-select-with-modal.js

```diff
@@ -34,7 +33,7 @@ import { filterOptions, defaultMatcher } from 'ember-power-select/utils/group-ut
  * @param {string} [subText] - Text to be displayed below the label
  * @param {string} [placeholder] - placeholder text to override the default text of "Search"
  * @param {function} onChange - The onchange action for this form field. ** SEE UTIL ** search-select-has-many.js if selecting models from a hasMany relationship
- * @param {string | Array} inputValue -  A comma-separated string or an array of strings -- array of ids for models.
```

Comment:
Is there somewhere where we accepted a comma-seperated string? I know in input fields we've removed tooltips that said we did (the cli allows it but we don't in those fields). Just want to make sure it's not legacy functionality we just don't use anymore.

Reply:
Yeah, I poked around before removing this documentation. The biggest indicator for me was the search select file had no way of handling a string. If a comma separated string was passed in, it didn't work because it couldn't map over it. Thanks for following up!